gnomad/sample_qc/pipeline.py

```diff
@@ -269,9 +284,22 @@ def annotate_sex(
         mt = mt.filter_rows(
             (hl.len(mt.alleles) == 2) & hl.is_snp(mt.alleles[0], mt.alleles[1])
         )
-    mt = hl.filter_intervals(
-        mt, [hl.parse_locus_interval(contig) for contig in x_contigs]
-    )
+
+    if is_vds:
```

Comment:
I think I'm not fully following this. For a VDS at this point `mt` is just the split biallelic variant data MT so `filter_intervals` should work fine on it right? And this only keeps the `mtds.variant_data`